start/src/test/java/org/apache/accumulo/start/util/AsyncSocketAppenderTest.java

```diff
@@ -36,7 +31,7 @@
 
   @Before
   public void setUp() throws Exception {
-    sa = createMock(SocketAppender.class);
+    sa = EasyMock.createMock(SocketAppender.class);
```

Comment:
Why make all of the changes in this file to remove the static imports? Seems unnecessary.